google/cloud/bigtable/mutations.h

```diff
@@ -319,6 +324,7 @@ class FailedMutation {
   /// @name accessors
   SingleRowMutation const& mutation() const { return mutation_; }
   grpc::Status const& status() const { return status_; }
+  google::cloud::Status const& gcstatus() const { return gcstatus_; }
```

Comment:
I think I was not specific enough.  I think this should be:

```C++
  google::cloud::Status const& status() const { return status_; }
  // no gcstatus() function
```

And we should change all the code to deal with that new API (yup, it would be a big change).

Reply:
So in FailedMutation class, remove the ```grpc::Status status_``` variable, its accessor and the constructor that takes in a ```google::rpc::Status``` and converts it into a  ```grpc::Status status_``` and the static function that does that
and  have only  a ```google::cloud::Status status_``` , its accessor and the constructor that takes in a   ```google::cloud::Status status_```, correct?

Reply:
Correct.